app/controllers/staff/proposal_reviews_controller.rb

```diff
@@ -35,7 +35,12 @@ def show
   end
 
   def update
-    authorize @proposal, :review?
+    if params[:program_mode] = true
```

Comment:
Hmm, this line seems suspect.  Isn't this always true as written, since you're doing assignment (= vs ==)?
Can you use the program_mode?  method here instead?